classes/cf-input-settings-types.class.php

```diff
@@ -9,7 +9,18 @@
 */
 class cfs_input_block extends cf_input_block {
 	function __construct( $conf ) {
-		return cf_input_block::cf_input_block($conf);
+		return cf_input_block::cf_input_block( $conf );
+	}
+
+	/**
+	 * Added support for php 5.3-
+	 *
+	 * @param $conf
+	 *
+	 * @return cf_input_block
+	 */
+	function cfs_input_block( $conf ) {
```

Comment:
Missing function visibility. I know it's `public` by default, we should make that explicit.